### ISSUE TITLE
Who will be on the initial steering council?

### ISSUE THREAD

**Issue by rabernat** [pangeo-data/governance#8]
While we finalize the governance docs, we need to face a few more specific questions:
- who will be on the first steering council?
- how many members should we aim for?

Some issues to consider in making this decision:
- initial council should reflect who are the de-facto leaders of the project now
- council members should show passion and enthusiasm for the Pangeo vision
- council members should be prepared to invest substantial time and energy in the project (no honorary members or silent partners)
- diversity of institutions (university, lab, industry, etc.)
- balance of institutions (not stacked with people from one place)
- capital D Diversity (this is the hard one, since our contributors are not very Diverse, and council members must come from contributors)

I'll offer an initial partial proposal. I think that at least myself, @jhamman, and @niallrobinson should be on the council. Beyond this, I am curious to hear what the community thinks.

**Comment by rabernat** [comment]
Any thoughts on this important topic? cc @pangeo-data/pangeo-admin @pangeo-data/pangeo-dev

**Comment by kmpaul** [comment]
I second @jhamman being on the council.  I believe that he can represent NCAR better than anyone else here.

**Comment by mrocklin** [comment]
It might be reasonable to rotate one person through the steering council who is intentionally new and not a core contributor.  Sometimes external perspectives are nice.  This slot might also make it easier to find people that satisfy the diversity constraint.

**Comment by mcgibbon** [comment]
To point at the elephant in the room, we need women (plural). I think "contributors" should be treated quite broadly - anyone who engages the geoscientific Python ecosystem by helping others in their institution find and use tools, or opens issues, or contributes to any geoscientific Python package, is qualified. In my view, constructing a steering council isn't about picking the N most involved people, it's about selecting a distribution of people who will reach and represent the concerns of as wide a group as possible.

I say this with only a loose understanding of what the "steering council" even is. Where is this defined?

**Comment by mcgibbon** [comment]
Is this an open call for nominations?

**Comment by darothen** [comment]
I'd like to contribute the idea that we try to include at least one PhD-level student on the steering council; because education and outreach is one of the pivotal community-building components we identified at the workshop, we need to make sure we have a strong ambassador to very-early career scientists.

**Comment by jhamman** [comment]
> I say this with only a loose understanding of what the "steering council" even is. Where is this defined?

@mcgibbon - See #1 and #5. (both of these are still pending comments from the community).

----

I also like the idea of including a younger participant in the steering council.

**Comment by jacobtomlinson** [comment]
I second @niallrobinson. I also recommend @rprudden could fill the PhD student slot as she is currently using our Pangeo for her PhD work.

**Comment by kmpaul** [comment]
Considering that our diversity is quite low at the moment, I would have no problem electing the current leaders of the community for a 2-year term (or similar) with the mandate to increase diversity during the first term.  That means emphasizing education and outreach as priorities for the steering council during the first term.

NCAR has resources that I think can be brought to bear on this problem.  The NCAR [Office of Diversity, Equity & Inclusion](https://www.ucar.edu/who-we-are/diversity-inclusion/office) could give a lot of guidance, I think.  I plan on reaching out to them this week, following on discussions from the dev meeting last week.

**Comment by kmpaul** [comment]
@mrocklin I think that's a fair statement.  I would be happy with shorter terms.

I was primarily thinking that 1-2 years seems like a rough time to spin a new person up to the point of being a community leader, depending on how much of their workload can be devoted to Pangeo work.  And this is what we are talking about when trying to get fresh _diverse_ blood into the community, I think.

**Comment by rabernat** [comment]
I agree it is very important to involve junior scientists and increase the diversity in Pangeo (I said this loud and clear in my recent [blog post](https://medium.com/pangeo/the-2018-pangeo-developers-workshop-1be359dac33c) and in my original issue above).
However, if you look at #5, which defines the role of the steering council, it clearly requires some degree of existing involvement and investment in the project. For example, the council will
> - Make decisions about the overall scope, vision and direction of the project
> - Make decisions about strategic collaborations with other organizations or individuals.

We need to figure out how to balance the need for experienced leadership with the need for broader community participation.

**Comment by kmpaul** [comment]
@rabernat I agree with the language of #5.  It seems to me that achieving this balance should be a first priority of the steering council, and that the steering council should initially be comprised of the current project leaders.  The steering council can push issues of outreach, education, and diversity efforts, such as tutorials and reaching out to diversity groups, such as PyLadies (mentioned by @yuvipanda).  

I guess what I realize I am suggesting is that we form a steering council based on what we know now, and let the steering council figure out how better to achieve its goals (and potentially modify the governance documents) over the next couple of years.  In other words, is it a good idea to move forward with a "minimum viable steering council" with the intent that we use this initial steering council to improve our governance and representation over time?

**Comment by mcgibbon** [comment]
At the initial meeting where Pangeo was formed, we identified a lack of representation of women as an issue. To this date, we have not made significant progress on that issue. Failing on this for so long speaks to the ability of our current leadership and membership to "make decisions about strategic collaborations with other organizations or individuals" and to push the issue of diversity efforts. I believe involving new community representatives, particularly women, is an important step towards making progress on that issue. This bucket has been kicked down the road for quite some time now. Thinking that it is acceptable to continue kicking that bucket speaks to the bias of our group, including myself - I haven't taken the time or effort or spent social capital to raise this issue before now.

Without laying blame on any one person, it's obvious that we as a group have bias in who we know and invite to collaborate and participate in Pangeo. You can see that by looking at who is participating. Without making a conscious effort to involve women, nothing will change. I would go as far as to suggest a required minimum of 2 women on the steering committee to counter our bias. I say 2 to avoid issues that arise from tokenism (it's difficult to assert your opinion as a token member), and to better reflect the community we serve. Fully capable, qualified, and involved women exist in our fields who are invested in the issues underlying this group. The members of this group by and large are participants because someone reached out to them inviting them to participate. We need to be inviting women.

I think the two bullets you chose to copy both highlight the need to have women on the steering council. Don't you think having women participate in "making decisions about strategic collaborations with other organizations or individuals" might significantly broaden the reach of the project?

The first bullet point in the original issue is "initial council should reflect who are the de-facto leaders of the project now", which means all white males, if I'm still up-to-date. The last bullet point is, to paraphrase "We need diversity (but it's really hard, because we're only going to consider this non-diverse pool of candidates)". I think we should re-consider whether it is a good idea to look at a non-diverse pool of candidates to select leadership for a group that is having an issue with diverse representation. I believe the pool can be widened without diluting the quality and suitability of candidates.

I'm specifically focusing on women above, but there are of course other axes along which we need to increase representation.

**Comment by mrocklin** [comment]
@mcgibbon I think that everyone agrees with you that increasing the diversity within the group is important to having a broader impact.  I think that people also feel that it is important that people deciding what happens to this community have demonstrated a strong understanding and commitment to the project.  Given the pool of currently active community members to choose from there is some tension here.  It seems to that balancing between those two objectives is a tricky question currently placed on this community.

From what you're saying it sounds like you lean strongly to one side, that it is critical to have at least two women in particular on the steering committee, even if they have little experience with the community and have not demonstrated commitment to the project.  That's a reasonable stance.  There are probably other reasonable stances on that spectrum between the two goals.  I think it's important to establish that this is a difficult problem, and that no single solution is correct.

> Without making a conscious effort to involve women, nothing will change.

For what it's worth, I've witnessed several of the people involved in this conversation make explicit and fully conscious efforts to increase diversity within this group.  This hasn't been as effective as they'd like, but it has been a constant focus of some individuals who lead this community.

Given your passion on this topic I might recommend that you join those individuals and work to help resolve the diversity problem by bringing in others to the community.

**Comment by mcgibbon** [comment]
@mrocklin that isn't quite an accurate characterization of my stance. I think that "experience with the community" should be interpreted more broadly than participation in Pangeo. Participation within one's institution to further goals that align with Pangeo's goals should count, as someone put in the (using and developing good tools, promoting the use of good tools, working with big data issues, presenting work in computing within our discipline). I think "commitment to the project" should be interpreted as "commitment to the goals of the project". This is how I would interpret what's [currently written in the guidelines](https://github.com/pangeo-data/governance/blob/council-membership/governance.md) for who should be on the council.

The one individual I brought to the project was a woman. Of the three others I talked to about participating in the project, none of them happened to be white males. My main inter-institutional collaboration is with a diverse individual on the project. I appreciate what others have done within the group. What we do as individuals does not negate the responsibility of the community to combat our structural bias, nor does it mean we are without significant bias as a group. I appreciate your suggestion and will continue to follow it, but let's not derail from the problem at hand by focusing on my personal responsibility.

The problem at hand is, we need diverse leadership and representation. We need more people in our leadership who have more research and social connections to diverse individuals. If we truly cannot find diverse candidates who would be suitable on the steering committee at this moment, we could write in the governing documents that there must be a minimum of 2 women on the steering committee by a date one year from now - if not, the steering committee would be reformed. That would ensure we actually follow up on getting women involved in this project. @kmpaul suggested something similar to this, I suggest we do it in a way that is enforceable with clear targets and consequence.

Like you and I have said, everyone has agreed that diversity is a problem we want to tackle. I do appreciate everyone here considering this issue to be important. Please also see that isn't enough on its own. For almost two years now, we haven't made progress. Let's make progress.

**Comment by chiaral** [comment]
I think that having a constructive and blunt conversation about this topic is very important, especially because we are at the very beginning of the process of creating the steering committee, so we have a chance to start with the right foot. This discussion should focus on the steps already taken into this direction and those we should take in the near future.

Short introduction for those who didn’t meet me: I am Chiara Lepore, I am a Research Scientist at Columbia, and a woman. I wasn’t at the meeting 2 years ago when Pangeo was formed. However I have been part of it since the more structured start with the Earth Cube funding. I am in fact one of the co-PIs of Pangeo.

Also, to frame my point of view, I am not an hacker. I have no interest in being a hacker, I am a scientist, that is why I am excited about this project. I can hopefully do things that before I would have not even dared to do, because I didn’t know how to compile/build or what-not complicated systems.  (On a less relevant note, I am also Ryan’s wife.)

Going back to my two main points, what did Pangeo actively do towards this direction of inclusion at its formal (proposal writing) inception?

Two (out of five) of the co-PIs from Columbia are women, my self and @naomi-henderson . Both of us are focused on the Use Cases. These become active, rather naturally, later in the life of a project as Pangeo.

The two most funded people on the project - that take more money out of the budget  (2.5 mo vs. 1 wk per year)- are indeed also us! the two women! We happen to be the most fragile figures, because we are soft money positions, and the project is recognizing that. 

At Lamont last spring Naomi and I led a Python course over two months. Ryan contributed to it as well in some advanced topics. We purposely wanted to teach python in a “friendly environment" (read as "not using any hacker language” and completely avoiding sentences like “you should open an issue” or “why don’t you ask on stackoverflow"). We covered all the building blocks of Pangeo, from installing conda and what is python, to Pandas, XArray, Dask, and developed many beginner and advanced notebooks (https://github.com/naomi-henderson/OCPhelp). Naomi did an excellent job and  literally hand held people through their first commit on Github. The group, about 20/30 people, was about 50% women.

If Pangeo were a typical NSF funded project I would say that it’s doing pretty well on the topic of diversity: the PIs are evenly distributed, money are very much supporting primarily minorities, and there has been a substantial effort in trying to include more diverse people in using Python and the Pangeo stack within Columbia.

Now, Pangeo is not a typical NSF project. It doesn’t end anymore with the PIs and coPIs people and has grown and keeps on growing at faster and faster rate. And yes, the community that has grown within this Github repo is definitely mostly men. However, it is important to point out that Pangeo is not only this GitHub repo.

So to my second point, what should we do next?

I was having a conversation with someone at the meeting - I am forgetting who - who was suggesting to work on making the contribution burden threshold lower so to allow for people to join the repo and contribute more easily. I am not sure that is entirely possible, because some technical steps to contribute to a GitHub repo are not avoidable, but mainly my answer was - I am not going to become a kubernetes hacker, I am good at what I do, which is science, and I will contribute to that part of the project. 
That part has tho less visibility on GitHub, although is there with the Use Cases, and we should make sure it gets more of visible in other ways and other places. Having a more diverse way to contribute to the project (not only code, but science as well), will, IMHO, create opportunities to reach a diverse audience as well.

Other step can be indeed including two women in the steering committee, but if I may express a completely personal and maybe controversial opinion, if we have two women in the steering committee but nothing changes at the community level, we are not solving the problem.

I personally think that some of the concerns we have about the premises of the steering committee are solved by this sentence already: 

> "When considering potential Members, the Council will look at candidates with a comprehensive view of their contributions. This will include but is not limited to code, code review, infrastructure work, mailing list and chat participation, community help/building, education and outreach, design work, etc. We are deliberately not setting arbitrary quantitative metrics (like “100 commits in this repo”) to avoid encouraging behavior that plays to the metrics rather than the project’s overall well-being. We want to encourage a diverse array of backgrounds, viewpoints and talents in our team, which is why we explicitly do not define code as the sole metric on which council membership will be evaluated.” 

I, in fact, really liked this sentence because I felt that a person with my background (atypical for a GitHub repo) could still become a council member! 

And I absolutely agree with what Kevin wrote and I think Jeremy and many others agree thus far, 

> I guess what I realize I am suggesting is that we form a steering council based on what we know now, and let the steering council figure out how better to achieve its goals (and potentially modify the governance documents) over the next couple of years. In other words, is it a good idea to move forward with a "minimum viable steering council" with the intent that we use this initial steering council to improve our governance and representation over time?

The very reason this issue was opened was to discuss what should be done, and that nothing is set in stone. I already see some constructive suggestions!

My constructive contribution therefore is two fold, and it’s essentially what others have suggested already. I am volunteering to kick off the effort in NYC on this, cannot tho commit on more than this for now: 1) we have an amazing diversity officer at Lamont, who has lots of expertise on diversity and inclusion within academia, I will reach out to her to get some inputs. These, tho, will not, most likely, be helpful for the “hacker world” but more for academia. 2) I (and Ryan) will reach out to Pyladies in NYC and set up a meetup to introduce Pangeo to their network. 

Each and every one of the people interested in the steering committee, or just on the topic of diversity and Pangeo, can do the same wherever they live!

On a lighter note, it’s great to see how this topic is discussed with so much passion among so many people!

**Comment by jacobtomlinson** [comment]
This is an excellent conversation and agree that the passion here is really good to see.

>  I will reach out to her to get some inputs. These, tho, will not, most likely, be helpful for the “hacker world” but more for academia.

I totally understand your reasons for the comment above, however as a member of the "hacker" community this is still something I am passionate to work towards. Technology does seem to be in a worse state than Science. Therefore if your diversity officer has any thoughts in that area, or contacts which may do, then I would be really keen to get their advice.

**Comment by niallrobinson** [comment]
Hi everyone - I've enjoyed reading through the discussion here.

Starting with diversity, I think there are some big bang-for-buck things we can do:

1. Change the website home page so that the first thing that people see below the logo is an invitation to join in. Diverse or not, that's unclear at the minute (if I find time I'll spin up a PR with @meganfitzsimons)

1. We should head-hunt/google and invite people from more diverse backgrounds to consider being involved. This is also a way to get try and get some good new collaborators!

1. We should make an effort to present at targeted community groups like PyLadies

More stuff to do after this, but I think this is relatively low-hanging fruit

@rprudden what do you think about representing the PhD community on the steering group?

@chiaral 

>cannot tho commit on more than this for now

does this mean that you can't find the time to be on the Steering Council? Shame ;)

-------

> how many members should we aim for?

I think that this needs to be few enough that a proper dialogue can be had - my feeling is that <= 6 people is optimal, especially as this will be happening on video chat most of the time.

------------

@mcgibbon 
> I think "commitment to the project" should be interpreted as "commitment to the goals of the project".

I'm a bit conflicted by this. Certainly, it will be valuable to have these people as part of the broader community, but I'm not sure about on the steering group. I think there is an argument that the steering group should represent people with a vested interest in Pangeo, as they can balance decisions against the amount of further work/investment they necessitate.

**Comment by rabernat** [comment]
Thanks everyone for this valuable discussion. I want to especially thank @mcgibbon for calling out some biased assumptions in my language and approach to the steering council membership. We as a community clearly have a lot to learn in this area.

With respect to the question at hand, I would suggest that we first clarify the roles and membership eligibility for the steering council in general terms before moving forward with selection of specific council members. The language it that document is mostly copied verbatim from Project Jupyter, but their choices may not be the right fit for Pangeo. It was premature of me to start this discussion before those questions were resolved. As suggested in #9, I have merged the initials PRs and have created #10 to formally seek comments on the [complete draft governance document](https://github.com/pangeo-data/governance/blob/master/governance_DRAFT.md) from the community. I encourage those of you who have strong thoughts on this to continue to participate in this discussion.

**Comment by mcgibbon** [comment]
Thank you for the informative post @chiaral, and for correcting some of my outdated knowledge about Pangeo.

> At Lamont last spring Naomi and I led a Python course over two months.

That sounds great! Reducing the barrier of entry to programming-related tasks is really important in allowing a wider group to be involved (which is a big part of why I'm working on [CliMT and Sympl](https://www.geosci-model-dev-discuss.net/gmd-2018-100/)). I'm glad to hear about the good performance of the grant with respect to diversity! The tasks you've taken upon yourself should indeed be a constructive contribution.

>> I think "commitment to the project" should be interpreted as "commitment to the goals of the project".

> I'm a bit conflicted by this. Certainly, it will be valuable to have these people as part of the broader community, but I'm not sure about on the steering group. I think there is an argument that the steering group should represent people with a vested interest in Pangeo, as they can balance decisions against the amount of further work/investment they necessitate.

@niallrobinson I am also conflicted by this. I do think it's important for the steering committee to be vested. However I think that those who are participating in projects adjacent to or affiliated with Pangeo still have a vested interest in Pangeo. I think that diversity is a higher priority than having all steering committee members with explicitly in-group experience, provided everyone is vested in the goals and plans to be involved. However, it also sounds like you don't need to look outside the in-group to find two women who are more than suitable for the council. I think it's appropriate to take that diversity from those who are most vested (already participating in the project) first.

As a PhD student, I'd be happy being represented by fellow machine learning fan @RPrudden.

>> cannot tho commit on more than this for now

> does this mean that you can't find the time to be on the Steering Council? Shame ;)

That doesn't sound like what @chiaral was saying. It also sounds to me like it would be great to have her on the council both for the merits of her participation and to have diversity of priorities and perspective. I'm surprised she wasn't mentioned earlier as a nominee.

I agree with what @chiaral was saying that having two women on the steering council doesn't solve the problem. I think it's an important step towards that goal, and to prevent a step in the wrong direction.

@rabernat it sounds like a good idea to finalize the membership eligibility and selection criteria before focusing on specific members. I'll look to comment there later. Thank you for directing the discussion towards that point!

**Comment by kmpaul** [comment]
Wow!  This is fantastic!  I came into my office this morning with a pile of work to do and an email inbox exploding with updates on this issue.  I love the passion on this, and I am very happy to see this conversation moving toward a "what can we do" theme, rather than a "how did we fail" theme.

@mcgibbon Thanks, very much, for your attention to this.  I like to think that I prioritize diversity, myself, but you correctly express the urgency on this issue, and I think we need to mobilize to address it now.  I hope you will be deeply involved in helping us improve on this front.

And since I agree with @chiaral's suggestion that we share what we have done on this front, I can say that I've been all-too slow to get the ball rolling on this front here at NCAR.  I have a team of 3 (all women) who are currently too busy with other tasks to bring on to Pangeo, despite my attempts to tantalize them with the awesomeness that is Pangeo!  I expect one of them to have more time to commit to Pangeo come 2019, after CMIP6 starts to wind down.  However, since I know they are all overloaded, I just recently hired a new SE I (from Rwanda) to work full-time on Pangeo, starting in October.

Mary Haley, here at NCAR, is the head of the NCL team, and she has expressed interest in working with the Pangeo community, which I think can help with diversity.  Would she make a good nomination for the Steering Council?

I agree this is still not enough, but sometimes we move at the lightning pace of bureaucracy!  

@rabernat: Personally, I don't think you were premature with this issue at all.  I think this was the right thing to do, in part because I believe that these questions are intrinsically related to the topic itself.  I think we just need to iterate a bit more before we can make a decision.

As for what I can do, I appreciate the suggestions that have already been made by everyone already.  I have reached out to NCAR's head of the Office of Diversity, Equity, and Inclusion (Carolyn Brinkworth), and she suggested that I contact the [Women Who Code](https://www.womenwhocode.com/) organization to find more involvement.  (There isn't a PyLadies branch in Boulder/Denver, but there is a Women Who Code branch.)

**Comment by jhamman** [comment]
This issue is getting stale. I'd like to wrap this up before all momentum is lost. Do we need to vote on initial committee members? A few have been suggested but I'm finding it hard to gauge the buy in on those proposed members.

**Comment by mcgibbon** [comment]
There still hasn't been an open call for nominations that I'm aware of. I asked above if we should be posting nominations here and wasn't answered, then it was said that we should clarify the documents before taking nominations. I would say that should be done for a period of 1-2 weeks and announced in all suitable channels, including github and the mailing list, so everyone feels welcome to nominate someone if they choose. Then we could vote with similar announcements.

**Comment by rabernat** [comment]
We need to accept the governance document before we can move forward with the council. But how do we accept the governance document without any leadership? No one has the "authority" to accept it. It's a catch 22!

I don't have any alternative to propose that will be satisfactory to everyone involved. So we are stuck in a stalemate.

**Comment by mcgibbon** [comment]
We could have a ratification vote with the final version sent to everyone on the mailing list. Possibly with an e-mail sent to the mailing list to propose changes before the vote. If everyone has the opportunity to propose changes and to voice whether the documents are acceptable, there shouldn't be any problems. 

I would also say though, I haven't seen any e-mails about this on the list, so giving people at least some time to provide input before the vote would be a good idea. This probably should have been e-mailed out much earlier - not everyone uses github, and github users aren't an even sample of pangeo members.

Does someone "in charge" (@rabernat?) think this is a good idea and want to do this?

**Comment by mrocklin** [comment]
If we were to have a vote I'm curious what the criteria would be for who gets to vote.

FWIW I would suggest folks circulate a proposal for comments and then @rabernat @jhamman @kmpaul @chiaral and @niallrobinson get together and just decide something by fiat, taking those comments into account.

**Comment by mcgibbon** [comment]
“Anyone signed up for the mailing list" is a common eligibility criteria.

**Comment by mcgibbon** [comment]
I should point out, "who gets to vote" is a problem that needs to be solved regardless of how we ratify these documents - we need a vote on the council members.

**Comment by mrocklin** [comment]
As someone who has invested a non-trivial amount of his recent professional career into this effort I'm against allowing the general public the ability to control this project.  I would prefer to see control go to people who have contributed to the project in a meaningful way.

**Comment by mcgibbon** [comment]
Usually it's all members of a community, not just "contributors", that vote in council members and ratify constitutional changes. That's the case for every academic group (e.g. student council, club) and organization (e.g. AMS, AGU) I've ever joined, at least. I agree the "general public" shouldn't be voting on this, but you need some way to define "members" of the group, and currently the only membership we have is github and mailing list membership. I agree that it would be better to have a better definition of member.

I would imagine anyone interested in or using big data working in the geosciences should be eligible to sign up for membership. The group may be defined differently than I would imagine, such that only contributors to Pangeo projects can be members - if that's the case, this should be clearly lined out in the governance documents. That would mean Pangeo represents a group of people working on a particular set of projects in the geosciences, rather than all members of the geoscientific community interested in and working with big data. Is this an umbrella group, or a closed group?

I would think the same voting mechanism should be used for ratifying the documents and for selecting council members.

**Comment by shoyer** [comment]
I think it's perfectly fine to bootstrap a (mostly) open group with a core
group of visionaries who sign the charter and determines the initial
leadershio. Then you can open up membership to the broader public who can
participate in due tune. Even then, a straight up vote for determining the
board may not be a great idea (e.g.,.consider NumFOCUS). In all
seriousness, I don't see any other way reasonable way to start a mission
driven organization. Otherwise you run the risk of losing sight of the
mission before you get started.

**Comment by chiaral** [comment]
Hello all, 

Can we maybe ask how other similar entities (although we all agree that they are not necessarily similar, but maybe similarly created?) solved this catch-22? I am thinking of Project Jupyter and the likes? I have to say tho that I am definitely ignorant about the history of those projects, so maybe they are not a good example. Not necessarily following their lead, but at least to have an idea of how things were handled. (We have one suggestion already, thanks @shoyer !) 

About the advertisement, we can/should re-advertise these github issues, I want to note tho that @rabernat  did it on twitter, https://twitter.com/rabernat/status/1034088046675873792 , he cc-ed the whole Pangeo github community (above) and then @kmpaul  opened another issue, https://github.com/pangeo-data/pangeo/issues/383  again to the whole Pangeo community.
We can definitely circulate this discussion on the mailing list, I didn't even know there was one, and maybe add that they can reply by mail if they don't have a github account.

The flaw that i see in @mcgibbon's  argument is the fact that for AGU and AMS there are fees to be paid to be a member. That fee is not to cover any cost about AGU and AMS,  it's too small, but it is essentially to filter out people from the "general population". It's a simple trick used anywhere to make sure that people that are going to have a say on certain thing are minimally involved. We of course do not want a paid fee, so I think the definition of membership is quite tricky, but it has to require some type of tangible involvement (in place of the monetary fee).

**Comment by mrocklin** [comment]
For reference I think that NumFOCUS emerged with a fixed set of people.  The recent board selection invited endorsements from community members who had either donated money or time to the organization.  Even then, these endorsements from community members weren't votes, and didn't decide who sits on the next board.  I'm sure that they were taken into account but in the end I suspect that it was a discussion among the current board and staff (though I'm not sure and have no inside knowledge here).

**Comment by rabernat** [comment]
The mailing list -- pangeo@googlegroups.com ([archive](https://groups.google.com/forum/#!forum/pangeo)) -- has fallen out of use as as a means of communication. I'm not sure why this happened--people just stopped sending emails to it. Consequently it is not mentioned on the [website](http://pangeo.io/contact.html). And it is out of sync with the [collaborators page](http://pangeo.io/collaborators.html#people). We have moved almost all communication to github.

I agree that the mailing list should have been notified (anyone could have done this), but I don't agree that this is an optimal way to define pangeo "membership" today.  In fact, the notion of "members" (apart from "council members") does not appear in the draft governance document at all. Borrowing from other open source projects, we only define "contributors".

> Usually it's all members of a community, not just "contributors", that vote in council members and ratify constitutional changes.

> I would think the same voting mechanism should be used for ratifying the documents and for selecting council members.

This is most definitely not what is in the current draft governance documents. @mcgibbon - is this a change you would now like to propose? If so, I guess you should make a PR. Clearly we do not have a consensus on this issue yet.

I would propose the following procedure, which tries to follow the actual draft governance plan as it currently stands as closely as possible (given the catch-22 of not having a council to start with):
- we notify the mailing list and wait another week for comments / PRs on the draft governance document
- we appoint with an initial temporary steering council of two people: @rabernat and @kmpaul (chosen because we are the PIs on the Pangeo NSF award, and thus already in a formal position of responsibility for this effort)
- this temporary council approves and merges the governance plan, which has hopefully reached a consensus
- the temporary council oversees a nomination process and appointment of a non-temporary steering council (as defined in the governance document), taking into account all the many issues regarding participation, diversity, inclusion, and representation that have been raised in this discussion. *We hope to achieve a consensus on the non-temporary steering council.* If not, the temporary steering council does their best to make a wise executive decision in selecting a non-temporary steering council.

**Comment by rabernat** [comment]
As a sort of straw poll, please feel free to click 👍 or 👎 on the above proposal.

**Comment by mcgibbon** [comment]
I don't mean to propose any changes to the structure of Pangeo, only to ask for clarification and say "this is what my perceptions were". I had thought of Pangeo as society focused (like PyLadies) rather than project focused (like Jupyter). Re-reading the [original mission statement](https://pangeo.io/about.html#mission-statement), it is more in-line with a project focus (at the time I had thought of "cultivate an ecosystem" as establishing a network of scientists, but in retrospect others meant cultivating a collection of software packages). There's nothing wrong with either configuration, and I don't have an opinion either way.

It sounds like the more closed and focused governance models are appropriate given the opinions people are presenting here.

**Comment by RPrudden** [comment]
> I had thought of "cultivate an ecosystem" as establishing a network of scientists, but in retrospect others meant cultivating a collection of software packages

I think it’s useful to draw a distinction between the specific Pangeo project, and the broader community exploring approaches to geoscience research related to big data and scalability. While I’d consider myself strongly invested in the latter, I’m only tangentially involved in the former. As such, I guess I don’t have a strong opinion on the organisation of the project steering committee.

**Comment by rabernat** [comment]
@RPrudden - it's no secret that you are very likely to be nominated for the steering council! I hope you will come to consider yourself involved in "Pangeo the Project," as we would really benefit from your perspective.

**Comment by rabernat** [comment]
@mcgibbon - you have strong opinions on Pangeo governance, and we value your input. On multiple occasions you have "called out" biases and other problematic issues with our approach Given your strong feelings, I really encourage you to participate constructively in this process. For example, we could really use your review of #16, our code of conduct. Comments on the overall governance structure are also welcome and encouraged. I hope that we can reach a consensus.

**Comment by chiaral** [comment]
> I had thought of Pangeo as society focused (like PyLadies) rather than project focused (like Jupyter). Re-reading the [original mission statement](https://pangeo.io/about.html#mission-statement), it is more in-line with a project focus (at the time I had thought of "cultivate an ecosystem" as establishing a network of scientists, but in retrospect others meant cultivating a collection of software packages). There's nothing wrong with either configuration, and I don't have an opinion either way at this point.

I just wanted to mention that I think what Jeremy wrote here helped clarifying some different perspectives throughout this thread (to myself as well!). So thanks for pointing this out!

**Comment by kmpaul** [comment]
With the above PR and the new interim steering council, I'm wondering if we should close this issue down and create a new issue for the nomination process?  It would make sense to keep everything in this issue, but it has gotten quite long.  Maybe start fresh?
